src/crimson/osd/object_context.h

```diff
@@ -146,6 +146,8 @@ class ObjectContext : public Blocker,
       return _with_lock(lock.excl_from_read(), std::forward<Func>(func));
     case RWState::RWEXCL:
       return _with_lock(lock.excl_from_excl(), std::forward<Func>(func));
+    case RWState::RWNONE:
```

Comment:
i am curious what's the use case of this function.

Reply:
it's used [here](https://github.com/ceph/ceph/blob/851d10ed3516a88586515913d95df514d065e53d/src/crimson/osd/replicated_recovery_backend.cc#L648)